catalog/java/io/material/catalog/slider/res/layout/cat_slider_fragment.xml

```diff
@@ -33,4 +33,11 @@
       android:value="8.09"
       android:valueFrom="0.0"
       android:valueTo="11.0" />
+
+  <com.google.android.material.button.MaterialButton
```

Comment:
nit, this should just be `Button` since `MaterialButton` will automatically be inflated. I can change this when I pull it in.